### ISSUE TITLE
Netflix doesn't work after last patch

### ISSUE THREAD

**Issue by netowww** [wltrsjames/Ultrawide-Video#3]
After the last patch in both of my computers that I have Ultrawide-Video extension, I am having problems to make it work. I tried pressing ctrl+alt+c and also tried clicking on enable in the Ultrawide-Video icon on the top bar of Chrome, but it simply doesn't work anymore (doesn't change nothing at all). Can you correct it?

**Comment by wltrsjames** [comment]
Thanks for the comment! I am aware of this, essentially Netflix have done an update, but not all of the users have got it yet. I had to patch for this update, not thinking about the people who haven't got it yet. Try pressing ctrl+shift+r to clear your cache and reload. If this doesn't work, my upcoming patch with fix this. I have found a way to get the extension working more generally, on most html5 players. Update will be out really soon
